exp-player/addon/components/exp-lookit-dialogue-page/component.js

```diff
@@ -607,6 +618,16 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
 
     },
 
+    parentTextStyle: Ember.computed('parentTextBlock', function() {
+        var parentTextBlock = this.get('parentTextBlock') || {};
+        var css = parentTextBlock.css || {};
+        var cssString = '';
+        $.each(css, function(propName, value) {
+            cssString += new Ember.Handlebars.SafeString(`${propName}:  ${value};`);
```

Comment:
See note elsewhere about `SafeString`. (which should more accurately named, `GeeIHopeThisIsASafeStringBecauseIAmChoosingToTrustIt`)